### PR TITLE
Let user know what's wrong when Web Bluetooth API is missing

### PR DESCRIPTION
**Pull request by sowbug**
Also make home page links relative to make it easier to do local
development, e.g., with `python -m SimpleHTTPServer`.

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Thank you @sowbug!
Here are some fixes that we might want to add to this PR before merging this: https://github.com/WebBluetoothCG/demos/commit/01abcaa43da919448242e0a5cc14d4259a4dac98?w=1

**Comment by sowbug** [comment]
Thanks, François. I added your suggestions to my fork.

File: index.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <h1>Web Bluetooth Demos</h1>
 <ul>
-  <li><a href="https://webbluetoothcg.github.io/demos/bluetooth-toy-plane/">bluetooth-toy-plane</a></li>
```
Comment (scheib) [inline index.html:4]: The full path is used to force https redirect when folks find http://webbluetoothcg.github.io/demos.

File: playbulb-candle/index.html [hunk 2]
```diff
@@ -14,7 +14,19 @@
   <body id="state">
     <div id="step1">
       <div id="title">PLAYBULB CANDLE DEMO</div>
-      <button id="connect" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">CONNECT</button>
+      <button id="connect" class="mdl-button mdl-js-button
+    mdl-button--raised mdl-js-ripple-effect mdl-button--accent"
```
Comment (scheib) [inline playbulb-candle/index.html:18]: Weird indent.

File: playbulb-candle/index.html [hunk 3]
```diff
@@ -14,7 +14,19 @@
...
+      style="display:none">CONNECT</button>
+
+      <div id="no-bluetooth" class="no-bluetooth-card-square mdl-card
+                                    mdl-shadow--2dp" style="display:none">
+        <div class="mdl-card__title mdl-card--expand">
+          <h2 class="mdl-card__title-text">No Web Bluetooth</h2>
+        </div>
+        <div class="mdl-card__supporting-text">
+          Please enable the Web Bluetooth API at chrome://flags/#enable-web-bluetooth
```
Comment (scheib) [inline playbulb-candle/index.html:27]: This demo should work for non-Chrome browsers too as they come up. So we should only mention chrome flags if we know we're on Chrome. We should point to the implementation status - as the platform and OS permissions matter too. https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md And that page includes flags information. Perhaps generic text such as:

Web Bluetooth isn't currently enabled. Please check that your browser and device support it and have any necessary development flags set.

**Comment by sowbug** [comment]
Feel free to adjust the wording. It'd make sense to land something soon while the traffic is still high from CDS.

File: index.html (hunk 1 above)
Comment (beaufortfrancois) [inline index.html:4]: Let's remove this part from this PR for now as it's only about Web Bluetooth API not being available.

File: playbulb-candle/index.html (hunk 2 above)
Comment (beaufortfrancois) [inline playbulb-candle/index.html:18]: I'll remove this and add custom style to `styles.css`.

File: playbulb-candle/index.html (hunk 3 above)
Comment (beaufortfrancois) [inline playbulb-candle/index.html:27]: Generic text will work for me.

**Comment by beaufortfrancois** [comment]
Based on @scheib feedback, here's a commit that @sowbug can add and we'll merge it: https://github.com/WebBluetoothCG/demos/commit/8b250d75813e924e610d3ab9aa2e3185d06ad4a2

**Comment by beaufortfrancois** [comment]
For info, here's what it looks like now:

screenshot 2015-11-24 at 10 42 30 am

**Pull request merged**